server.js

```diff
@@ -1,248 +1,229 @@
+      if (err) throw err;
+      let departments = res.map(department => ({ name: department.department_name, value: department.department_id }));
+      connection.query(`SELECT * FROM employee;`, (err, res) => {
+        if (err) throw err;
+        let employees = res.map(employee => ({ name: employee.first_name + ' ' + employee.last_name, value: employee.employee_id }));
         inquirer.prompt([
+          createPrompt('firstName', 'input', "What is the new employee's first name?"),
+          createPrompt('lastName', 'input', "What is the employee's last name?"),
+          createPrompt('role', 'rawlist', "What is the employee's title?", roles),
+          createPrompt('dept', 'rawlist', 'What department is the new employee in?', departments),
+          createPrompt('salary', 'number', "What is the new employee's salary?"),
+          createPrompt('manager', 'rawlist', "Who is the new employee's manager?", employees)
+        ]).then((response) => {
+          connection.query(`INSERT INTO employee SET ?`,
             {
-            name: 'title',
-            type: 'input',
-            message: 'What is the name of the role you want to add?'   
-            },
-            {
-            name: 'salary',
-            type: 'input',
-            message: 'What is the salary of the role you want to add?'   
-            },
-            {
-            name: 'deptName',
-            type: 'rawlist',
-            message: 'Which department do you want to add the new role to?',
-            choices: departments
+              first_name: response.firstName,
+              last_name: response.lastName,
+              role_id: response.role,
+              manager_id: response.manager,
             },
-        ]).then((response) => {
-            connection.query(`INSERT INTO role SET ?`, 
+            (err, res) => {
+              if (err) throw err;
+            })
+          connection.query(`INSERT INTO role SET ?`,
             {
-                title: response.title,
-                salary: response.salary,
-                department_id: response.deptName,
+              department_id: response.dept,
             },
             (err, res) => {
-                if (err) throw err;
-                console.log(`\n ${response.title} successfully added to database! \n`);
-                startApp();
+              if (err) throw err;
+              console.log(`\n ${response.firstName} ${response.lastName} successfully added to database! \n`);
+              startApp();
             })
         })
+      })
     })
+  })
 };
 
-addAnEmployee = () => {
-    connection.query(`SELECT * FROM role;`, (err, res) => {
-        if (err) throw err;
-        let roles = res.map(role => ({name: role.title, value: role.role_id }));
-        connection.query(`SELECT * FROM department;`, (err, res) => {
+function updateEmployeeRoles() {
+  connection.query(`SELECT * FROM role;`, (err, res) => {
```

Comment:
@samrogers15 Welcome to [callback hell!](http://callbackhell.com/) Code like this is the reasons promises came about in JS in the first place. A couple things I would suggest here:

1. You're submitting a couple queries to your db that don't depend on each other at all. Rather than wait for the first one to come back before sending the second one, it'd be more efficient (and easier to manage via less nested code) by using `Promise.all` to send both requests simultaneously and then proceed once they've both returned data
2. Once you have that data available in your first response block, you can then refactor to use `.then` chaining